internal/overlord/servstate/handlers.go

```diff
@@ -216,12 +215,10 @@ func (m *ServiceManager) doStop(task *state.Task, tomb *tomb.Tomb) error {
 			// Stopped successfully.
 			return nil
 		case <-tomb.Dying():
-			// Stop operation cancelled (ignore for simplicity).
-			logger.Noticef("Ignoring cancellation of stop for service %q", request.Name)
-		case <-timeout:
-			// Should never happen, because killTimeElapsed and exited both send to the
-			// "stopped" channel, but don't block in case we got something wrong.
-			return fmt.Errorf("internal error: timed out waiting for stop")
+			// User tried to abort the stop, but SIGTERM and/or SIGKILL have
+			// already been sent to the process, so there's not much more we
+			// can do than log it.
+			logger.Noticef("Cannot abort stop for service %q, kill signals already sent", request.Name)
```

Comment:
Either s/signals/signal/ or or s/kill//, I guess?

Reply:
Thanks - I updated to the latter.